master/usage/migrating-pools.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Migrating Active Pools
```

Comment:
Sounds a bit like it's the pool that you're migrating.  How about "Migrating to a new IP pool"?